src/platform/code-editor/code-editor.component.ts

```diff
@@ -616,5 +668,7 @@ export class TdCodeEditorComponent implements OnInit, AfterViewInit, ControlValu
     window.addEventListener('resize', () => {
         this._editor.layout();
     });
+    this.addFullScreenModeCommand();
   }
+
 }
```

Comment:
please also update the two public methods in the API: documentation Readme:
https://github.com/Teradata/covalent-code-editor/blob/develop/docs/API.md

Reply:
I made all functions as private as it is not required for end user.
Those functions will be executed when user click on Full Screen in context menu and press ESC if already in full screen.